Dockerfile

```diff
@@ -19,6 +20,19 @@ RUN apt-get update && \
       python-scapy \
       tcpdump
 
+# Clone and build behavioral-model, which is needed by some tests.
+ENV BEHAVIORAL_MODEL_REPO https://github.com/sethfowler/behavioral-model.git
```

Comment:
I should have read your explanation too. Go ahead and merge and we'll switch to main repo when Antonin merges 277.